inc/core/admin.php

```diff
@@ -86,92 +84,120 @@ public function incompatibility_notice() {
 		if ( $current_screen->id === 'appearance_page_neve-welcome' ) {
 			return;
 		}
-
+		$theme_name  = apply_filters( 'ti_wl_theme_name', $this->theme_args->__get( 'Name' ) );
 		$plugin_name = apply_filters( 'ti_wl_plugin_name', 'Neve Pro' );
 
 		$notifications = [];
-		$theme_slug    = 'neve';
-		$themes_update = get_site_transient( 'update_themes' );
-		if ( isset( $themes_update->response[ $theme_slug ] ) ) {
-			$update                       = $themes_update->response[ $theme_slug ];
-			$notifications[ $theme_slug ] = [
-				'type' => 'theme',
-				'path' => '',
-				/* translators: %s - theme name */
-				'cta'  => sprintf( __( 'Update %1$s to v%2$s', 'neve' ), $this->theme_name, $update['new_version'] ),
-			];
+		if ( current_user_can( 'update_themes' ) ) {
+			$theme_slug    = 'neve';
+			$themes_update = get_site_transient( 'update_themes' );
+			if ( isset( $themes_update->response[ $theme_slug ] ) ) {
+				$update                       = $themes_update->response[ $theme_slug ];
+				$notifications[ $theme_slug ] = [
+					'type' => 'theme',
+					'path' => '',
+					/* translators: %s - theme name */
+					'cta'  => sprintf( __( 'Update %1$s to v%2$s', 'neve' ), $theme_name, $update['new_version'] ),
+				];
+			}
 		}
 
-		$plugins_update = get_site_transient( 'update_plugins' );
-		$plugin_path    = 'neve-pro-addon/neve-pro-addon.php';
-		if ( isset( $plugins_update->response[ $plugin_path ] ) ) {
-			$update                          = $plugins_update->response[ $plugin_path ];
-			$notifications['neve-pro-addon'] = [
-				'type' => 'plugin',
-				'path' => $plugin_path,
-				/* translators: %s - pro plugin name (Neve Pro) */
-				'cta'  => sprintf( __( 'Update %1$s to v%2$s', 'neve' ), $plugin_name, $update->new_version ),
-			];
+		$pro_data = get_option( 'neve_pro_addon_license_data' );
+		$valid    = isset( $pro_data->license ) && $pro_data->license === 'valid';
+
+		if ( current_user_can( 'update_plugins' ) && $valid ) {
```

Comment:
Checks that the current user can update plugin and that the license is valid. 

Only then, the plugin is added to the notice.